### PR TITLE
DevOps Role

### PR DESCRIPTION
**Pull request by homeyer**

### CODE REVIEW

File: CAREERS.md [hunk 1]
```diff
@@ -46,4 +48,30 @@ We use our own product to manage our work. Anyone in the world can see what we
...
+#### About You
+You have deep technical expertise in containers and distributed systems. You are passionate about working with customers. You possess strong engineering skills, and have built software using nodeJS.
+
+#### Signs of an Awesome Candidate
+- You have experience with Docker, CoreOS, Kubernetes or similar
+- You have experience in cloud architecture and deployments
+- You’re able to think on your feet to debug customers issues in realtime
+- You are resourceful at debugging systems and networks with common *NIX tools
+- You understand and have worked with microservice architectures
+- You have knowledge of DNS, CDN, load balancing, firewalls, VPNs, SMTP, SSL, etc. — all the things that make the web work.
```
Comment (songawee) [inline CAREERS.md]: May already be implied, maybe add HTTP?

File: CAREERS.md [hunk 2]
```diff
@@ -46,4 +48,30 @@ We use our own product to manage our work. Anyone in the world can see what we
...
 ## Open Positions
 
-_We don't have any open positions at this time. If you would one day love to work with the Waffle team, reach out! We'd love to meet you._
+### DevOps Engineer
+
+#### Description
+At Waffle, we develop a micro service architecture that’s deployed as a SaaS product and also shipped behind the firewall for our enterprise version. Our on-premises delivery is state of the art, built with Docker components running on Replicated. We’re looking for a new teammate who can help us continue to build out this architecture, support our on-premises customers as we start to scale, and also contribute towards scaling our cloud infrastructure.
+
+#### About You
+You have deep technical expertise in containers and distributed systems. You are passionate about working with customers. You possess strong engineering skills, and have built software using nodeJS.
```
Comment (songawee) [inline CAREERS.md]: Could we update the last word `nodeJS` to `NodeJS` or `Node.js`?

File: CAREERS.md [hunk 3]
```diff
@@ -46,4 +48,30 @@ We use our own product to manage our work. Anyone in the world can see what we
...
+
+#### Signs of an Awesome Candidate
+- You have experience with Docker, CoreOS, Kubernetes or similar
+- You have experience in cloud architecture and deployments
+- You’re able to think on your feet to debug customers issues in realtime
+- You are resourceful at debugging systems and networks with common *NIX tools
+- You understand and have worked with microservice architectures
+- You have knowledge of DNS, CDN, load balancing, firewalls, VPNs, SMTP, SSL, etc. — all the things that make the web work.
+- You are a troubleshooter at heart and enjoy diving in the deep end, figuring things out, and solving problems.
```
Comment (songawee) [inline CAREERS.md]: could remove the periods at the end of these last three lines to match up with items earlier in the list

**Comment by burtonjc** [comment]
I agree with Dennis' suggestions, other than that 👍

**Pull request merged**